PURL-TYPES.rst

```diff
@@ -81,15 +81,24 @@ conan
 -----
 ``conan`` for Conan C/C++ packages:
 
-- The default repository is ``https://center.conan.io``
-- The ``namespace`` is the user if present
-- The ``name`` is the package name.
-- The ``version`` is the package version.
-- The qualifier ``channel`` must be not empty if namespace is present
+- The default repository is ``https://center.conan.io``.
+- The ``namespace`` is the vendor.
+- The ``name`` is the Conan package name.
+- The ``version`` is the Conan package version.
+- The qualifier ``channel`` is the Conan channel, it is required if the Conan package was published with channel.
+- The qualifier ``user`` is the Conan user, it is required if the Conan package was published with user.
```

Comment:
"with user" -> "with *a* user" / "as a user" / "under a user"?

Reply:
same as for channel